File: Python/sorting_algorithms/BubbleSort.py
def BubbleSort(list):
    for i in range(0, len(list)):
        for j in range(0, len(list)-i-1):
            if list[j] > list[j+1]:
                list[j], list[j+1] = list[j+1], list[j]

    return list

File: C#/SortingAlgorithms/BubbleSort.cs
namespace SortingAlgorithms;

class BubbleSort
{
    public static int[] sort(int[] list)
    {
        for (int i = 0; i < list.Length; i++)
        {
            for (int j = 0; j < list.Length - i - 1; j++)
            {
                if (list[j] > list[j + 1])
                {
                    int curVal = list[j];
                    list[j] = list[j + 1];
                    list[j + 1] = curVal;
                }
            }
        }
        return list;
    }
}